### PR TITLE
DM-29569: PTC maxMeanSignal no longer works 

### PR DESCRIPTION
**Pull request by plazas**

### CODE REVIEW

File: python/lsst/cp/pipe/ptc/cpSolvePtcTask.py [hunk 1]
```diff
@@ -231,6 +231,10 @@ def run(self, inputCovariances, camera=None, inputExpList=None):
                     datasetPtc.rawVars[ampName].append(partialPtcDataset.rawVars[ampName][0])
                 else:
                     datasetPtc.rawVars[ampName].append(partialPtcDataset.rawVars[ampName])
+                if type(partialPtcDataset.expIdMask[ampName]) is list:
```
Comment (czwa) [inline python/lsst/cp/pipe/ptc/cpSolvePtcTask.py:234]: Are there any cases where these are not lists with a single element each?

File: python/lsst/cp/pipe/ptc/cpSolvePtcTask.py [hunk 2]
```diff
@@ -607,7 +611,12 @@ def errFunc(p, x, y):
                 assert (len(mask) == len(timeVecOriginal) == len(meanVecOriginal) == len(varVecOriginal))
             if not (mask.any() and newMask.any()):
                 continue
-            dataset.expIdMask[ampName] = mask  # store the final mask
+            dataset.expIdMask[ampName] = np.array(dataset.expIdMask[ampName])
```
Comment (czwa) [inline python/lsst/cp/pipe/ptc/cpSolvePtcTask.py:614]: I'm sure the answer is "yes", but these are mask==1 is good, correct?

**Review by czwa — APPROVED** [review]

File: python/lsst/cp/pipe/ptc/cpSolvePtcTask.py (hunk 1 above)
Comment (plazas) [reply python/lsst/cp/pipe/ptc/cpSolvePtcTask.py:234]: In theory, no. After running {{cpExtract}}, these should be lists with single elements.

File: python/lsst/cp/pipe/ptc/cpSolvePtcTask.py (hunk 2 above)
Comment (plazas) [reply python/lsst/cp/pipe/ptc/cpSolvePtcTask.py:614]: More precisely, they have `True` (correct) and `False`

**Pull request merged**